shared_types/exercise.ts

```diff
@@ -1,7 +1,10 @@
+import { scamCategories } from './scamCategory';
```

Comment:
```suggestion
import type { scamCategories } from './scamCategory';
```

Reply:
Oh true thanks ! I need to take the habit ^^